org.mixxx.Mixxx.yaml

```diff
@@ -42,7 +42,7 @@ modules:
   - shared-modules/libmad/libmad.json
   - shared-modules/python2.7/python-2.7.json
   - shared-modules/gudev/gudev.json
-  - shared-modules/linux-audio/lv2.json
+  - shared-modules/linux-audio/jack2.json
```

Comment:
I am confused... doesn't this need PipeWire?

Reply:
The pipewire in the SDK doesn't have the header for jack. This build jack for the headers and just remove it completely, so that pw-jack is used.

Reply:
PipeWire is now optionally shipping the JACK header so that should be fixed.

Reply:
As I said, it's not in the current Flatpak KDE Sdk.